quipucords/api/scanjob/tests_scanjob.py

```diff
@@ -328,6 +402,56 @@ def test_connection_failed_success(self):
 
         self.assertEqual(json_response, expected)
 
+    def test_connection_name_ordering(self):
+        """Get ScanJob connection results for systems ordered by name."""
+        # pylint: disable=no-member
+        scan_job, scan_task = create_scan_job(
+            self.source, ScanTask.SCAN_TYPE_INSPECT)
+
+        # Create two connection system results one failure & one success
+        sys_result = SystemConnectionResult(name='Foo',
+                                            source=self.source,
+                                            credential=self.cred,
+                                            status=SystemConnectionResult
+                                            .SUCCESS)
+        sys_result2 = SystemConnectionResult(name='Bar',
+                                             source=self.source,
+                                             credential=self.cred,
+                                             status=SystemConnectionResult
+                                             .FAILED)
+        sys_result.save()
+        sys_result2.save()
+        conn_result = scan_task.prerequisites.first().connection_result
+        conn_result.systems.add(sys_result)
+        conn_result.systems.add(sys_result2)
+        conn_result.save()
+
+        url = reverse('scanjob-detail', args=(scan_job.id,)) + 'connection/'
+        url += '?ordering=-name'
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        json_response = response.json()
+        expected = {'count': 2,
+                    'next': None,
+                    'previous': None,
+                    'results': [
+                        {'name': 'Foo',
```

Comment:
So this is supposed to be ordering these items on the name value? Why would `Foo` come before `Bar`?

Reply:
I guess you are saying that descending is `Z X Y ... A` instead of `A B C D....`?

Reply:
correct `-name` means sort the strings in reverse alphabetical order.